std/typecons.d

```diff
@@ -5108,7 +5108,7 @@ private static:
             {
                 preamble ~= "alias self = " ~ name ~ ";\n";
                 if (WITH_BASE_CLASS && !__traits(isAbstractFunction, func))
-                    preamble ~= "alias parent = AliasSeq!(__traits(getMember, super, \"" ~ name ~ "\"))[0];";
+                    preamble ~= "alias parent = __traits(getMember, super, \"" ~ name ~ "\");";
```

Comment:
Looks like a bug in the new feature: `std/typecons.d-mixin-4429(4442,30): Error: expected 1 function arguments, not 0`

Reply:
gotta wait for D-Scanner to parse TypeTraits anyway so doubly [blocked] (label as, please)

Reply:
Done.